test/public/runs/runsPerSimulationPass.overview.test.js

```diff
@@ -128,7 +128,7 @@ module.exports = () => {
         await reloadPage(page);
         await expectLink(page, 'tr#row56 .column-ITS a', {
             href: 'http://localhost:4000/?page=qc-flags-for-simulation-pass&runNumber=56&dplDetectorId=4&simulationPassId=2',
-            innerText: '0!',
+            innerText: '0',
```

Comment:
You should probably still look for the icon to be there

Reply:
done